specification/azurefleet/AzureFleet.Management/common.tsp

```diff
@@ -1,19 +1,44 @@
 import "@typespec/http";
 import "@typespec/rest";
 import "@typespec/versioning";
-import "@azure-tools/typespec-autorest";
 import "@azure-tools/typespec-azure-core";
 import "@azure-tools/typespec-azure-resource-manager";
 
 using TypeSpec.Http;
 using TypeSpec.Rest;
 using TypeSpec.Versioning;
-using Autorest;
 using Azure.ResourceManager;
 using OpenAPI;
 
 namespace Microsoft.AzureFleet;
 
 @doc("ApiError for Fleet")
-@useRef("../../../../../compute/resource-manager/Microsoft.Compute/common-types/v1/common.json#/definitions/ApiError")
-model ApiError {}
+model ApiError is ApiErrorBase {
+  @extension("x-ms-identifiers", ["message", "target"])
+  @doc("The Api error details")
+  details?: Array<ApiErrorBase>;
+
+  @doc("The Api inner error")
+  innererror?: InnerError;
+}
+
+@doc("Api error base.")
+model ApiErrorBase {
+  @doc("The error code.")
+  code?: string;
+
+  @doc("The target of the particular error.")
+  target?: string;
+
+  @doc("The error message.")
+  message?: string;
+}
+
+@doc("Inner error details.")
+model InnerError {
+  @doc("The exception type.")
+  exceptionType?: string;
+
+  @doc("The internal error message or exception dump.")
```

Comment:
>@doc("The internal error message or exception dump.") [](http://example.com/codeflow?start=2&length=53)

This description suggest we return internal error and exception details which would not be an information disclosure risk, sounds like not a good security practice

Reply:
We are using the same ApiError which [Microsoft.Compute](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json#L63) uses currently hence keeping it the same.